src/Autocomplete.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo, useState } from 'react';
+import cn from 'classnames';
+import { debounce } from 'lodash';
+import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { DropdownMenu } from './DropdownMenu';
+import { DropdownInput } from './DropdownInput';
+import { Deley } from './types/Deley';
+
+type Props = {
+  deley: Deley;
```

Comment:
https://github.com/mate-academy/react_autocomplete/pull/218#discussion_r1328456241 - not fixed 😢